### PR TITLE
Copy toast

### LINKED ISSUE #269: Add toast for when you press copy

### PR DESCRIPTION
**Pull request by benalleng**
Should close #269. I also added a cursor pointer class to the copy icon svg to just drive home it is interactable UI just in case.

### CODE REVIEW

**Comment by benalleng** [comment]
> Preview URL: https://mutiny-testnet-iz0disa43-paulteam.vercel.app

Neat! fastest place to find a copy is to make a receive for 0 sats

File: frontend/src/components/Copy.tsx [hunk 1]
```diff
@@ -1,4 +1,6 @@
+import { toast } from "react-hot-toast"
 import { ReactComponent as CopyIcon } from "../images/icons/copy.svg"
+import MutinyToaster from "./MutinyToaster"
```
Comment (futurepaul) [inline frontend/src/components/Copy.tsx:3]: this means there will be two mutinytoasters on some pages... but it doesn't seem to cause double toasts so no harm.

we can probably end up with one global mutinytoaster at some point but this is fine for now

File: frontend/src/images/icons/copy.svg [hunk 2]
```diff
@@ -1 +1 @@
-<svg fill="none" aria-label="Copy" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 2h11v2H6v13H4V2zm4 4h12v16H8V6zm2 2v12h8V8h-8z" fill="currentColor"/></svg>
\ No newline at end of file
+<svg class="cursor-pointer" fill="none" aria-label="Copy" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 2h11v2H6v13H4V2zm4 4h12v16H8V6zm2 2v12h8V8h-8z" fill="currentColor"/></svg>
```
Comment (futurepaul) [inline frontend/src/images/icons/copy.svg:1]: I would've added this class to the wrapper div not the svg, but no biggie

**Review by futurepaul — APPROVED** [review]
sweet, love having the preview url!!

**Pull request merged**